### PR TITLE
New column on user/list and new order on user/edit

### PR DESCRIPTION
**Pull request by cahedral**
I have created a new column in user list form and I order the user form too

### CODE REVIEW

**Comment by cahedral** [comment]
I added too, other commit

File: upload/admin/language/en-gb/localisation/language.php [hunk 1]
```diff
@@ -31,6 +31,7 @@
 $_['error_name']        = 'Language Name must be between 1 and 32 characters!';
 $_['error_code']        = 'Language Code must be between 2 and 5 characters!';
 $_['error_locale']      = 'Locale required!';
+$_['error_status']      = 'All languages are inactived. You must active one of them';
```
Comment (TheCartpenter) [inline upload/admin/language/en-gb/localisation/language.php:34]: ```$_['error_status']      = 'All languages are inactive. You must activate one of them';```

Comment (cahedral) [reply upload/admin/language/en-gb/localisation/language.php:34]: Thanks

**Comment by danielkerr** [comment]
no need

**Pull request closed without merging**